lib/perf_hooks.js

```diff
@@ -467,6 +469,10 @@ function doNotify() {
 // Set up the callback used to receive PerformanceObserver notifications
 function observersCallback(entry) {
   const type = mapTypes(entry.entryType);
+
+  if (type === NODE_PERFORMANCE_ENTRY_TYPE_HTTP2)
+    collectStats(entry);
```

Comment:
I would prefer we didn't define these here, but rather in the HTTP2 part. I think these are also defined for every single instance, which will increase overhead.

Reply:
@mcollina Can you comment that on @jasnell’s #18020? :)